### ISSUE TITLE
Error loading pretrained weigths

### ISSUE THREAD

**Issue by EnricoBeltramo** [MVIG-SJTU/WSHP#20]
I have follow error when I try to load pretrained networks in inference.py:

DataLossError (see above for traceback): Unable to open table file D:\Progetti\Tensorflow\GAN\WSHP\parsing_network\models\final_model\checkpoint: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
         [[node save/RestoreV2 (defined at D:\Progetti\Tensorflow\GAN\WSHP\parsing_network\inference.py:109) ]]
         [[node save/RestoreV2 (defined at D:\Progetti\Tensorflow\GAN\WSHP\parsing_network\inference.py:109) ]]

**Comment by Fang-Haoshu** [comment]
Maybe the tensorflow version is not correct?

**Comment by EnricoBeltramo** [comment]
I'm using tensorflow gpu 1.13 on Windows 10. Should be a problem? What is the reference version?

**Comment by Fang-Haoshu** [comment]
we use tf 1.1.0 on linux

**Comment by brunomsantiago** [comment]
A little bit late, but I was having the same issue (Python 2.7, tensorflow 1.1.0).  Found the answer on [stack overflow](https://stackoverflow.com/questions/41048819/how-to-restore-a-model-by-filename-in-tensorflow-r12) by user Anggi Permana Harianja.
Use:
`models/final_model/model.ckpt-19315`
instead of:
`models/final_model/model.ckpt-19315.data-00000-of-00001`
